Resources/Prototypes/secret_weights.yml

```diff
@@ -1,8 +1,30 @@
 - type: weightedRandom
   id: Secret
-  weights:
+  weights: #these are the presets!!! NOT gamerules.
     Nukeops: 0.20
     Traitor: 0.50
     Zombie: 0.10
     Revolutionary: 0.20
-    
\ No newline at end of file
+
+- type: weightedRandom
+  id: Secret_MRP
+  weights:
+    Nukeops: 0.15
+    Traitor: 0.40
+    Zombie: 0.10
+    Revolutionary: 0.10
+    Extended: 0.25
+
+- type: weightedRandom
+  id: Secret_LRP
+  weights:
```

Comment:
Just call it SecretGameMode as it's the default (and to avoid prototype ID clashes).

Reply:
Should i just replace the Secret prototype at the top?

Reply:
Calling a default of something LRP is silly when it's just the default.